src/pages/Hyun Yong Jeong/Login/Login.scss

```
@@ -0,0 +1,84 @@
+@import "/Users/boikun/Desktop/wecode/wecode-react-intro/public/variables.scss";
```

절대경로로 해당 scss파일을 가져오는것이 아닌 상대경로로 파일을 import해 오는 것이 더 일반적입니다.
또한 variables.scss는 public폴더하위에 생성하는 파일이 아닙니다. 초기세팅 가이드를 참고해서 variables.scss가 어떤위치에 있어야 하는지 다시 한 번 확인 해보세요!
```suggestion
@import "../variables.scss";
```